spec/models/user_spec.rb

```diff
@@ -302,6 +159,19 @@
         to raise_error(ActiveRecord::RecordInvalid,
                        /Validation failed: Password is too long \(maximum is 128 characters\)/)
     end
+
+    it 'works with spaces' do
+      PASSWORD_W_SPACES = 'this has a few spaces'.freeze
+
+      prototype_user = create(:user)
```

Comment:
It's not necessary to create 2 users for this test. You can remove the prototype user, and update the user below to not specify an email since one will be created for it automatically via the Factory.